apps/mobile/src/components/Feed/CommentsBottomSheet/CommentsBottomSheet.tsx

```diff
@@ -114,6 +142,15 @@ export function CommentsBottomSheet({
     return isSubmittingComment;
   }, [isSubmittingComment, isSubmittingPostComment, type]);
 
+  const handleReplyPress = useCallback((params: OnReplyPressParams) => {
+    setSelectedComment(params);
+    commentBoxRef.current?.focus();
+  }, []);
+
+  const inputPlaceholder = useMemo(() => {
+    return `Replying to ${selectedComment?.username ?? 'comment'}`;
```

Comment:
let's keep the default placeholder as "Add a comment" instead of "Replying to comment"